src/time.jl

```diff
@@ -23,13 +23,9 @@ function every(dt)
     n
 end
 
-function weakrefdo(ref, yes, no=()->nothing)
```

Comment:
I spent some time looking, but I confess I don't really understand the design of `fps` well enough to be of any use.